Pages/navigation.html

```diff
@@ -94,8 +94,30 @@
         </div>
         <p>
           For reference, you can have a look at this website's nav bar on your
-          left.
+          left or here's an example below.
         </p>
+        <div
+          style="
+            position: unset;
```

Comment:
avoid inline css instead us class

Reply:
Actually, this is one-off use of the navbar, as by default navigation bar is fixed and this was for documentation as you can see in the snapshot. so creating a class for this might not be idle as there's no reuse of this code.